kafka_exporter.go

```diff
@@ -411,81 +446,111 @@ func (e *Exporter) collect(ch chan<- prometheus.Metric) {
 					topicPartitionLeader, prometheus.GaugeValue, float64(broker.ID()), topic, strconv.FormatInt(int64(partition), 10),
 				)
 			}
+			ch <- prometheus.MustNewConstMetric(
+				topicPartitions, prometheus.GaugeValue, float64(len(partitions)), topic,
```

Comment:
why set `topicPartitions` again ? it already set in line 434.

Reply:
error during code rebase. thanks.